connect-example/src/main/java/com/stripe/android/connect/example/ui/settings/SettingsViewModel.kt

```diff
@@ -117,38 +141,12 @@ class SettingsViewModel @Inject constructor(
 
     // Private functions
 
-    private fun loadSettings() {
+    private fun loadOneTimeSettings() {
         _state.update { state ->
-            val accountsFromService = embeddedComponentService.accounts.value ?: emptyList()
```

Comment:
👀 This is the key issue: we're calling `loadSettings()` once which only fetches the current value of `embeddedComponentService.accounts`. The fix is to observe the `embeddedComponentService.accounts` flow.